jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibExtension.java

```diff
@@ -185,7 +184,7 @@ public void setAllowInsecureRegistries(boolean allowInsecureRegistries) {
     this.allowInsecureRegistries.set(allowInsecureRegistries);
   }
 
-  public void setExtraDirectory(File extraDirectory) {
+  public void setExtraDirectory(Path extraDirectory) {
```

Comment:
Does this work for `build.gradle`? I remember we used `File` originally because `Path` doesn't work with the Gradle project `file` method.

Reply:
Oh I didn't check, I thought an integration test would have covered this already. One sec.